public/auth.php

```diff
@@ -1,4 +1,23 @@
 <?php
 require_once 'ably.php';
+// Check if the user is logged in
+if (isset($_COOKIE['username'])) {
+    // Issue a token request with pub & sub permissions on all channels +
+    // configure the token with an identity
+    $tokenParams = [
+        'capability' => [
+            '*' => ['publish', 'subscribe']
+        ],
+        'clientId'  => $_COOKIE['username']
```

Comment:
formatting error, i was aiming to align both `=>` in same column, i'll fix it in second version